lib/multigrid.cpp

```diff
@@ -660,7 +664,7 @@ namespace quda
       param_coarse_solver->verbosity_precondition = param.mg_global.verbosity[param.level+1];
 
       // preconditioned solver wrapper is uniform precision
-      param_coarse_solver->precision = r_coarse.Precision();
+      param_coarse_solver->precision = r_coarse[0].Precision();
```

Comment:
iirc we added support for directly calling `*.Precision()` on the ColorSpinorVector wrapper?

Reply:
Indeed we did, but the vectors contained in multigrid.cpp are not `quda::vector_ref<ColorSpinorField>`, and instead they are `std::vector<ColorSpinorField>`.

Eventually we could replace all use of `std::vector` with the QUDA derived flavor, `quda::vector`, but that's quite a bit of surgery, that is beyond the scope of this PR.